package.json

```diff
@@ -70,7 +78,9 @@
     "nodemon": "^2.0.20",
     "prettier": "^2.4.0",
     "pretty-quick": "^3.1.3",
+    "readline-sync": "^1.4.10",
```

Comment:
This is an archived project. We should try to find an alternate library for example. Please remove it from here.